modules/account-lib/package.json

```diff
@@ -43,6 +43,7 @@
     "@bitgo/sdk-coin-rbtc": "^1.1.2",
     "@bitgo/sdk-coin-sol": "^2.0.0",
     "@bitgo/sdk-coin-stx": "^1.2.0",
+    "@bitgo/sdk-coin-sui": "^1.0.0",
```

Comment:
new coin is not going to be in account-lib

Reply:
removed from account-lib